sharktank/tests/models/t5/t5_test.py

```diff
@@ -498,19 +493,19 @@ def testCompareAgainstTransformers(
 
         theta = Theta(
             {
-                "attn_q.weight": DefaultPrimitiveTensor(
+                "q.weight": DefaultPrimitiveTensor(
```

Comment:
What is the motivation with all the renames? Is it just the corresponding names from huggingface's layout instead of gguf's?

Reply:
Yes.